package desafio.one.digitalinnovation.personapi.exception;

public class PersonNotFoundExecpition extends Exception {

    public PersonNotFoundExecpition(){
        super("Entidade n√£o encontrada");
    }
}
